test/Chainweb/Test/Pact/TransactionTests.hs

```diff
@@ -250,52 +245,6 @@ testCoinbaseEnforceFailure = do
     logger = newLogger neverLog ""
 
 
-testCoinbaseUpgradeDevnet :: Assertion
-testCoinbaseUpgradeDevnet = do
-    (pdb,mc) <- loadScript "test/pact/coin-and-devaccts.repl"
-    r <- try $ applyCoinbase v logger pdb miner 0.1 pubData devnetHeader
-      creationTime (EnforceCoinbaseFailure True) (CoinbaseUsePrecompiled False) mc
-    case r of
-      Left (e :: SomeException) -> assertFailure $ "upgrade coinbase failed: " ++ (sshow e)
-      Right (T2 cr mcm) -> case (_crLogs cr,mcm) of
-        (_,Nothing) -> assertFailure "Expected module cache from successful upgrade"
-        (Nothing,_) -> assertFailure "Expected logs from successful upgrade"
-        (Just logs,_) -> do
-          void $ matchLogs logs
-            [("USER_coin_coin-table","abcd",Just 0.1)
-            ,("SYS_modules","fungible-v2",Nothing)
-            ,("SYS_modules","coin",Nothing)
-            ,("USER_coin_coin-table","sender07",Just 998662.3)
-            ,("USER_coin_coin-table","sender09",Just 998662.1)]
-  where
-    matchLogs logs logTests
-      | length logs /= length logTests =
-          assertFailure $ "matchLogs: length mismatch " ++ show (length logs) ++
-          " /= " ++ show (length logTests)
-      | otherwise = zipWithM matchLog logs logTests
-    matchLog log' (domain,key',balanceM) = do
-          assertEqual "domain matches" domain (_txDomain log')
-          assertEqual "key matches" key' (_txKey log')
-          case balanceM of
-            Nothing -> return ()
-            Just bal ->
-              assertEqual "balance matches" (Just (Number bal))
-                (preview (_Object . ix "balance") (_txValue log'))
-    v = Development
-    miner = Miner (MinerId "abcd") (MinerKeys $ mkKeySet [] "<")
-    pubData = PublicData def blockHeight' (toInt64 blockTime) ""
-    upgradeTime = fromJuste $ upgradeCoinV2Date v
-    blockTime = add (TimeSpan (Micros (- 1000))) upgradeTime
-    creationTime = BlockCreationTime $ add (TimeSpan (Micros 1000)) upgradeTime
-    toInt64 (Time (TimeSpan (Micros m))) = m
-    blockHeight' = 123
-    logger = newLogger neverLog "" -- set to alwaysLog to debug
-    devnetHeader = setBlockTime blockTime $ someBlockHeader v (BlockHeight blockHeight')
```

Comment:
We're removing this because now the `PublicData` won't determine the upgrades, yeah? I'm okay with this as long as we get Stuart to also sign off on this and we all 👍

Reply:
yes, we could write a test for the new code, which would be different and much simpler. However, we test this each time we do a mainnet catchup, this gets covered. Also we now have CI tests for validating against the actual mainnet history. So, I don't think we need this test any more.

Reply:
Let's keep it please for future reference when we need to test remediations.